starters/gatsby-wordpress-starter/__tests__/data/pageTemplateData.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
The `pagesIndex` and  singular `page` now get their data in different structures from one another in separate queries.